cicd/gitops/configuring-openshift-login-for-argo-cd.adoc

```diff
@@ -0,0 +1,26 @@
+[id="configuring-openshift-login-for-argo-cd"]
+= Configuring OpenShift login for Argo CD
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: configuring-openshift-login-for-argo-cd
+
+toc::[]
+
+Argo CD allows you to log in by using your OpenShift Credentials through Keycloak as an Identity Broker.
+
+.Prerequisites
+* Red Hat SSO is installed on the cluster.
+
+include::modules/gitops-creating-a-new-client-in-keycloak.adoc[leveloffset=+1]
+
+include::modules/gitops-configuring-argo-cd-oidc.adoc[leveloffset=+1]
+
+include::modules/gitops-keycloak-identity-brokering-with-openshift-oauthclient.adoc[leveloffset=+1]
+
+include::modules/gitops-registering-an-oauth-client.adoc[leveloffset=+1]
+
+include::modules/gitops-enabling-dex.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Both links point to external resources we will need Rick's approval for both.

Reply:
For now, I've commented them out until we get the approval.